### ISSUE TITLE
OpenRecord for the EditableGrid

### ISSUE THREAD

**Issue by NicolasPrats** [Microsoft/EasyRepro#400]
Hello,

The method `Microsoft.Dynamics365.UIAutomation.Api.UCI.OpenRecord `is working fine with readonly grids but not with editable grids.
The method is looking for rows with attribute `data-lp-id` but in the editable grids, the rows don't have this attribute.

**Comment by dtu11** [comment]
Yes, this is a gap in the API's as the editable grids are a little different as you point out.   I think we have a backlog item for this but if not i'll add it.

**Comment by markcunninghamuk** [comment]
I added a Highlight row as a workaround, you can the click the Edit command if it helps
